sorc/emcsfc_ice_blend.fd/emcsfc_ice_blend.f90

```diff
@@ -110,15 +101,21 @@ program emcsfc_ice_blend
  integer, parameter     :: imax=4320 !< Constant value on i dim.
  integer, parameter     :: jmax=2160 !< Constant value on j dim.
 
- integer                :: i,j, istat, iunit
+ integer                :: i,j, istat, iunit !< j search position of a file
  integer                :: ii, iii, jj, jjj, count
- integer                :: lugi
- integer                :: jdisc, jgdtn, jpdtn, k
+ integer                :: lugi !< set to 0 - no grib index file
+ integer                :: jdisc, jgdtn, jpdtn, k 
```

Comment:
You need to break the declaration of each var onto it's own line.

```
integer :: jdisk !< set to 2 - search for discipline
integer:: jgdtn !<  search for product definition template number
integer::jptn !< search for grid definition template number; 0 - lat/lon grid
integer:: k !< index
```

Reply:
Updated